web/src/routes/(user)/search/+page.svelte

```diff
@@ -43,7 +43,8 @@
 		}
 	});
 
-	$: term = $page.url.searchParams.get('q') || data.term || '';
+	$: clip_status = $page.url.searchParams.get('clip') !== 'false';
+	$: term = `${clip_status ? '' : 'm:'}${$page.url.searchParams.get('q')}` || data.term || '';
```

Comment:
What happens if you get a query like `/search?clip=true`? The search bar will just look like `m:`. Is this what we want?

Reply:
It'd stay empty. But yeah, `?clip=false` would populate an empty `m:` .

Reply:
That doesn't seem to pass validation.

![image](https://github.com/immich-app/immich/assets/9749173/b30ab331-6eb5-40f5-961d-35faadea273e)

Reply:
Which means this edgecase input is already not possible. It isn't possible to send an input such that the resulting page would have `clip=false` and an empty query.

Reply:
If I were to navigate to `https://example.com/search?clip=false` then it would produce `m:` which isn't valid,

Reply:
Gotcha. Fixed.